tests/queries/0_stateless/02706_show_columns.reference

```diff
@@ -45,3 +45,7 @@ uint64	UInt64	NO	PRI SOR	\N
 int32	Int32	NO		\N	
 str	String	NO		\N	
 uint64	UInt64	NO	PRI SOR	\N	
+--- SHOW COLUMNS FROM table with dots
+int32	Nullable(Int32)	YES		\N	
```

Comment:
You can add a heading of the query here because this test will lose its point without it. (look at 02071_lower_upper_utf8_row_overlaps)

Reply:
Sorry, I'm not sure I understand.
What heading should I add? I think that the line you are commenting is in the middle of the query output.

Reply:
It is the first line of the query output result. But the full query output contains the heading:

```
SHOW COLUMNS FROM test     <------ heading

Query id: 2e82db08-bf08-4493-96aa-fbea20a54a16

   ┌─field─┬─type──┬─null─┬─key─────┬─default─┬─extra─┐
1. │ a     │ Int64 │ NO   │ PRI SOR │ ᴺᵁᴸᴸ    │       │
   └───────┴───────┴──────┴─────────┴─────────┴───────┘
```

As the table name in the heading was also incorrect, we need to make sure that it is correct there after fix

Reply:
Do you mean that I should `-- { echoOn }` and `-- { echoOff }` around the query to make it print the query itself to check that it is correct?
That is not how echoOn/Off work. It just echos the query as it was written, it will always be "correct". The heading you see in the client is not part of the server response, it's just something that the client itself will generate by parsing and reformatting the query. It was incorrect because the parsing code was incorrect.
I could add and explicit test for the formatting code via formatQuery or something like that, but the formatting code was already correct.

Did I misunderstand what you mean?

Reply:
Yes, you seem to be right😃, I think this test will be OK

Reply:
Thanks!